content/src/content/jcr_root/apps/core/wcm/components/sharing/v1/sharing/amp.html

```diff
@@ -0,0 +1,26 @@
+<!--/*~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+  ~ Copyright 2019 Adobe
+  ~
+  ~ Licensed under the Apache License, Version 2.0 (the "License");
+  ~ you may not use this file except in compliance with the License.
+  ~ You may obtain a copy of the License at
+  ~
+  ~     http://www.apache.org/licenses/LICENSE-2.0
+  ~
+  ~ Unless required by applicable law or agreed to in writing, software
+  ~ distributed under the License is distributed on an "AS IS" BASIS,
+  ~ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  ~ See the License for the specific language governing permissions and
+  ~ limitations under the License.
+  ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~*/-->
+<sly data-sly-use.template="core/wcm/components/commons/v1/templates.html"
+     data-sly-use.socialMedia="com.adobe.cq.wcm.core.components.models.SocialMediaHelper"
+     data-sly-call="${template.placeholder @ isEmpty=!socialMedia.socialMediaEnabled}"></sly>
+
+<sly data-sly-test="${socialMedia.facebookEnabled && socialMedia.facebookAppId}">
```

Comment:
@gaetanmarmasse - If I understand correctly, this is a new configuration. I don't see it exposed in the the `SocialMediaHelper`, perhaps I'm missing it?

Reply:
You are right, It is missing. 
Looks like it got removed when we did the rebase from `development`.
I'll update.

Reply:
Should be addressed by https://github.com/adobe/aem-core-wcm-components/pull/861/commits/554e99c40fac68ee06af4e5cddb942bcb213e44b.